.gitignore

```diff
@@ -4,3 +4,4 @@ dist
 .env.heroku
 .vscode/last.sql
 .vscode/settings.json
+/pastebin-frontend
```

Comment:
What does this refer to? Did you have your front-end files in a local backend repo?